assets/js/modules/adsense/components/setup/v2/SetupCreateAccount.js

```diff
@@ -16,6 +16,115 @@
+import { CORE_SITE } from '../../../../../googlesitekit/datastore/site/constants';
+import { CORE_USER } from '../../../../../googlesitekit/datastore/user/constants';
+import { ErrorNotices, UserProfile } from '../../common';
+const { useSelect } = Data;
+
 export default function SetupCreateAccount() {
-	return <div>TODO: UI to create new account</div>;
+	const viewContext = useContext( ViewContextContext );
+	const eventCategory = `${ viewContext }_adsense`;
+	const siteURL = useSelect( ( select ) =>
+		select( CORE_SITE ).getReferenceSiteURL()
+	);
+	const userEmail = useSelect( ( select ) => select( CORE_USER ).getEmail() );
+	const existingTag = useSelect( ( select ) =>
+		select( MODULES_ADSENSE ).getExistingTag()
+	);
+	const signUpURL = useSelect( ( select ) =>
+		select( MODULES_ADSENSE ).getServiceCreateAccountURL()
+	);
+	const supportURL = useSelect( ( select ) =>
+		select( CORE_SITE ).getGoogleSupportURL( {
+			path: '/adsense/answer/2659101',
+		} )
+	);
+
+	const createAccountHandler = useCallback(
+		async ( event ) => {
+			event.preventDefault();
+			await trackEvent( eventCategory, 'create_account' );
+			global.open( signUpURL, '_blank' );
+		},
+		[ signUpURL, eventCategory ]
+	);
+
+	if ( ! siteURL || ! userEmail || undefined === existingTag ) {
+		return null;
+	}
+
+	return (
+		<Fragment>
+			<h3 className="googlesitekit-heading-4 googlesitekit-setup-module__title">
+				{ __( 'Create your AdSense account', 'google-site-kit' ) }
+			</h3>
+
+			<ErrorNotices />
+
+			<p>
+				{ __(
+					'Once you create your account, Site Kit will place AdSense code on every page across your site. This means your site will be automatically optimized to help you earn money from your content.',
+					'google-site-kit'
+				) }
+			</p>
+
+			<UserProfile />
+
+			<div className="googlesitekit-setup-module__action">
+				<Button onClick={ createAccountHandler } href={ signUpURL }>
+					{ __( 'Create AdSense account', 'google-site-kit' ) }
+				</Button>
+			</div>
+
+			<p className="googlesitekit-setup-module__footer-text">
+				{ existingTag &&
+					sprintf(
+						/* translators: 1: client ID, 2: user email address, 3: account ID */
+						__(
+							'Site Kit detected AdSense code %1$s on your page. We recommend you remove that code or add %2$s as a user to the AdSense account %3$s.',
+							'google-site-kit'
+						),
+						existingTag,
+						userEmail,
+						parseAccountID( existingTag )
+					) }
+				{ ! existingTag &&
+					sprintf(
+						/* translators: %s: user email address */
+						__(
+							'Already use AdSense? Add %s as a user to an existing AdSense account.',
+							'google-site-kit'
+						),
+						userEmail
+					) }{ ' ' }
+				<Link
+					href={ supportURL }
+					inherit
+					external
+					aria-label={ __(
+						'Learn more about adding a user to an existing AdSense account',
+						'google-site-kit'
+					) }
+				>
+					{ __( 'Learn more', 'google-site-kit' ) }
+				</Link>
```

Comment:
I know there are a lot of places where we use this sort of "space between two translated strings" approach, but technically it's not quite right. It presents problems when trying to localise this whole section of text properly as they are really one paragraph, but being presented to translators as two distinct pieces of text AND being combined without proper thought given to how a particular locale would combine them. Really we should be doing this:

```suggestion
					createInterpolateElement(
						sprintf(
							/* translators: %s: user email address */
							__(
								'Already use AdSense? Add %s as a user to an existing AdSense account. <a>Learn more</a>',
								'google-site-kit'
							),
							userEmail,
						),
						{
							a: <Link
								href={ supportURL }
								inherit
								external
								aria-label={ __(
									'Learn more about adding a user to an existing AdSense account',
									'google-site-kit'
								) }
							/>
						}
					)
```

I know it's sort of annoying, but it's the right approach here to give a really solid i18n experience.